doc/rados/operations/user-management.rst

```diff
@@ -218,10 +218,10 @@ namespace such that reads and writes by the user take place only within the
 namespace. Objects written to a namespace within the pool can only be accessed
 by users who have access to the namespace.
 
-.. note:: Currently (i.e., ``firefly``), namespaces are only useful for 
-   applications written on top of ``librados``. Ceph clients such as block 
-   device, object storage and file system do not currently support this 
-   feature.
+.. note:: namespaces are primarily useful for applications written on top of
+   ``librados`` where the logical grouping can alleviate the need to create
+   different pools. Ceph radosgw (from ``luminous``) uses namespaces for various
+   metadata objects.
```

Comment:
capitalize `Namespaces` at the beginning of the sentence, and `Ceph radosgw` -> `Ceph Object Gateway`